src/index.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter&family=Prata&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <div class="product-card__image">
+        <img
+          class="product-card__image-size"
+          src="./images/imac.jpeg"
+          alt="iMac">
+      </div>
+
+      <h1 class="product-card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="product-card__code reset">
+        Product code: 195434
+      </p>
+
+      <div class="stars-box">
+        <div class="stars-box__stars">
+          <div class="stars-box__star"></div>
+
+          <div class="stars-box__star"></div>
+
+          <div class="stars-box__star"></div>
+
+          <div class="stars-box__star"></div>
+
+          <div class="stars-box__star"></div>
+        </div>
```

Comment:
please check this part. and in your Stars task. it seems you've missed modifier in block with stars
![image](https://user-images.githubusercontent.com/31005050/193422782-6c533830-e6de-454d-a71e-2ebed86c684f.png)

Reply:
Thank you for your review! 
But here i don't actually agree with you.
According to checklist we can change names of block stars elements, but have to change it correctly.
I don't think that it make sense to name block elements modifier with number if we have only one block from stars blocks

Reply:
up to you ;)
it's just my advice to fix like they ask in checklist (with using modifier)

Reply:
i got it) thank you
Some checklist conditions don't absolutely clear for me
So let see what mentor will say))